config/services/dataplane_v1beta1_openstackdataplaneservice_ovn.yaml

```diff
@@ -7,3 +7,6 @@ spec:
   playbook: osp.edpm.ovn
   configMaps:
   - ovncontroller-config
+  secrets:
+  - neutron-external-config
+  - nova-metadata-compute-config
```

Comment:
This is correct in our default deployment as we deploy as single NovaMetadata there.

In a different deployment where the Nova part of OpenStackControlPlane is configure do deploy metadata per cell this needs to be adjusted to use the cell level config secret where the cell matches with the cell used in the name of the secret of the OpenStackDataPlaneService representing the nova-compute service.

I think we need to document his for the end user

Reply:
@gibizer what's the right place to document it? here in default service definition? in docs repo?

Reply:
It probably needs to exist in both nova and dataplane docs. I'll be working on updating the dataplane docs around the composable services interface, and we can add it there.

Reply:
The nova-operator side of this documentation is living in https://github.com/openstack-k8s-operators/nova-operator/pull/515/commits/827c51abc7f4a1cf4f3e06ecd7b3217d77185bf4

Reply:
https://github.com/openstack-k8s-operators/nova-operator/pull/515 is merged now but the name of the secret is changed to nova-metadata-neutron-config